src/main/scala/org/constellation/storage/SnapshotService.scala

```diff
@@ -54,7 +54,7 @@ class SnapshotService[F[_]: Concurrent](
       accepted <- acceptedCBSinceSnapshot.get
       lastHeight <- lastSnapshotHeight.get
       hashes = dao.snapshotHashes
-      addressCacheData <- addressService.toMap()
+      addressCacheData <- addressService.dump
```

Comment:
I think that `toMap` is more verbose than `dump`, especially if it's used not for debugging only